include/swift/ABI/Task.h

```diff
@@ -71,6 +71,8 @@ class alignas(2 * alignof(void*)) Job :
   // Derived classes can use this to store a Job Id.
   uint32_t Id = 0;
 
+  void *Reserved[2];
```

Comment:
I don't immediately see where this storage is zeroed?  (That can be important if you ever have different versions of this code trying to interoperate; new code using those fields needs to know when to ignore them.)

Reply:
Added zero initialization for this.